qiita_db/support_files/patches/67.sql

```diff
@@ -0,0 +1,2 @@
+alter table qiita.software_command add addtl_processing_cmd varchar;
```

Comment:
Could you follow the style of previous patches? Like the date and the goal of the changes are missing, thanks.

Reply:
Not a problem, thanks for pointing it out!